exasol/bucketfs/_buckets.py

```diff
@@ -27,6 +33,18 @@ class BucketLike(Protocol):
     It is compatible with both on-premises an SaaS BucketFS systems.
     """
 
+    @property
+    def name(self) -> str:
+        """
+        Returns the bucket name.
+        """
+
+    @property
+    def udf_path(self) -> str:
```

Comment:
should we maybe return a PosixPath

Reply:
BucketLike uses strings everywhere else. I think this property should also return a string for the sake of consistency.

Reply:
ok, fine by me